07_system_overview.R

```diff
@@ -128,495 +310,465 @@ system_df_enrl_flags <- system_df_prep %>%
+              AmIndAKNative +
+              Asian +
+              MidEastNAfrican +
+              White +
+              HispanicLatinaeo +
+              BlackAfAmerican == 0, 1, 0),
+  NativeHIPacificLatineExclusive1 =
+    if_else(NativeHIPacific == 1 & HispanicLatinaeo == 1 &
+              AmIndAKNative +
+              Asian +
+              MidEastNAfrican +
+              White +
+              BlackAfAmerican == 0, 1, 0),
+  WhiteAloneExclusive1 =
+    if_else(White == 1 &
+              AmIndAKNative +
+              Asian +
+              MidEastNAfrican +
+              NativeHIPacific +
+              HispanicLatinaeo +
+              BlackAfAmerican == 0, 1, 0),
+  WhiteLatineExclusive1 =
+    if_else(White == 1 & HispanicLatinaeo == 1 &
+              AmIndAKNative +
+              Asian +
+              MidEastNAfrican +
+              NativeHIPacific +
+              BlackAfAmerican == 0, 1, 0),
+  MultipleNotLatineExclusive1 =
+    if_else(HispanicLatinaeo == 0 &
+              AmIndAKNative +
+              Asian +
+              MidEastNAfrican +
+              NativeHIPacific +
+              White +
+              BlackAfAmerican > 1, 1, 0),
+  MultipleLatineExclusive1 =
+    if_else(HispanicLatinaeo == 1 &
+              AmIndAKNative +
+              Asian +
+              MidEastNAfrican +
+              NativeHIPacific +
+              White +
+              BlackAfAmerican > 1, 1, 0),
+  RaceEthnicityNoneExclusive =
+    if_else(
+      HispanicLatinaeo +
+        AmIndAKNative +
+        Asian +
+        MidEastNAfrican +
+        NativeHIPacific +
+        White +
+        BlackAfAmerican == 0,
+      1,
+      0
+    ),
+  # Data quality column to check for mutual exclusivity
+  DQExclusive1RaceEth =
+    AmIndAKNativeAloneExclusive1 +
+    AmIndAKNativeLatineExclusive1 +
+    AsianAloneExclusive1 +
+    AsianLatineExclusive1 +
+    BlackAfAmericanAloneExclusive1 +
+    BlackAfAmericanLatineExclusive1 +
+    LatineAloneExclusive1 +
+    MENAAloneExclusive1 +
+    MENALatineExclusive1 +
+    NativeHIPacificAloneExclusive1 +
+    NativeHIPacificLatineExclusive1 +
+    WhiteAloneExclusive1 +
+    WhiteLatineExclusive1 +
+    MultipleNotLatineExclusive1 +
+    MultipleLatineExclusive1 +
+    RaceEthnicityNoneExclusive, # all should equal 1
+  # exclusive logic group 2
+  BILPOCExclusive2 = if_else(
+    AmIndAKNative +
+      Asian +
+      MidEastNAfrican +
+      NativeHIPacific +
+      HispanicLatinaeo +
+      BlackAfAmerican > 0, 1, 0
+  ),
+  WhiteExclusive2 = if_else(
+    White == 1 &
+      AmIndAKNative +
+      Asian +
+      MidEastNAfrican +
+      NativeHIPacific +
+      HispanicLatinaeo +
+      BlackAfAmerican == 0, 1, 0
+  ),
+  # Data quality check for exclusive group 2
+  DQRaceEthExclusive2 =
+    BILPOCExclusive2 +
+    WhiteExclusive2 +
+    RaceEthnicityNoneExclusive, # all rows should equal 1
+  # inclusive logic group 1
+  AmIndAKNativeInclusive1 = if_else(AmIndAKNative == 1, 1, 0),
```

Comment:
At the very least, I don't think we need these Inclusives or we could simply rename the original variables. Mutations can lower performance. I get that naming the "Inclusive" makes it clear they are the parallel versions of the Exclusive ones. But the only difference between these and the originals are that the NAs have been replaced with 0s, and I THINK their only purposes are (1) creating the Exclusive variables, and (2) applying the user-specified filters, i.e. when you later do something like `if_any(.cols = c(input$syso_race_ethnicity)`. If we simply used the original versions that have NAs in them, I think that should be fine for both purposes, since, ultimately the important thing is that if_any function, which only reads 1s. It shouldn't matter if the values are NA.